apps/payments/next/app/[locale]/[offeringId]/[interval]/location/en.ftl

```diff
@@ -1,3 +1,5 @@
-location-required-header = Please select your country and enter your postal code.
-location-required-subheader = We weren’t able to detect your location automatically.
+location-header = Select your country and enter your postal code
+#productName (String) - The name of the product to create subscription, e.g. Mozilla VPN
+location-subheader = to continue to checkout for { $productName }
```

Comment:
```suggestion
location-header = Select your country and enter your postal code
#productName (String) - The name of the product to create subscription, e.g. Mozilla VPN
location-subheader = to continue to checkout for { $productName }
```

This won't work for localization. This should either be one string, or two complete sentences. 
Other languages have different word order and can't replicate what the English is doing here.

Reply:
Thanks for the reminder! I have updated the string, please let me know if it works for localization. Thank you!